Automattic/jetpack
# Jetpack AI: Use usage helper to set current tier value

Part of #33792.

## Proposed changes:

* Use the Jetpack AI Usage Helper to set the `current-tier.value` on the the `ai-assistant-feature` response, so it's ready when the new plans come alive

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* We need to check if the response is still the same as before
* Apply this change to your sandbox and sandbox the `public-api` endpoint
* Go to the WordPress.com Console and do a request to the `wpcom/v2` `/sites/$wpcom_site/jetpack-ai/ai-assistant-feature` endpoint:
* If the site used for the test has the paid plan, you should see `current-tier.value = 1`:

<img width="600" alt="Screenshot 2023-10-31 at 15 22 11" src="https://github.com/Automattic/jetpack/assets/6760046/5f242acd-8fe1-4cb8-b5bd-f0d5311d381a">

* If the site used for the test does not have the paid plan, you should see `current-tier.value = 0`:

<img width="600" alt="Screenshot 2023-10-31 at 15 23 04" src="https://github.com/Automattic/jetpack/assets/6760046/6f02955b-d89a-4f41-9507-31d673145b11">
